lametro/models.py

```diff
@@ -487,6 +487,21 @@ class LAMetroEvent(Event, LiveMediaMixin, SourcesMixin):
     class Meta:
         proxy = True
 
+    @classmethod
+    def most_recent_past_meetings(cls):
+        '''
+        Returns meetings in the current month that have occured in the past
+        two weeks.
+        '''
+        yesterday = datetime.today() - timedelta(days=1)
+        current_month = datetime.today().month
+        two_weeks_ago = datetime.today() - timedelta(weeks=2)
+
+        past_meetings = cls.objects.filter(
+            Q(start_time__month=current_month), Q(start_time__range=(two_weeks_ago, yesterday))).order_by('start_time')
```

Comment:
Can you say more about ending the range yesterday? What happens if a meeting happed at 10 a.m., and now it's 1 p.m.?

Reply:
BTW, TIL you can pass a range! So much nicer than `__gt` and `__lt`. Are both ends of the range inclusive?

Reply:
When I wrote this, I was assuming that we wouldn't want to display events from earlier in the current day. But IIRC we received clarification that we do want to include these events. I can modify this.

And yes, both ends of `range` are inclusive! From the Django docs:
<img width="844" alt="Screen Shot 2022-03-09 at 3 33 14 PM" src="https://user-images.githubusercontent.com/36973363/157531048-defdfdf8-28d5-434d-b95b-de0689957333.png">